dog-checkout-page4.html

```diff
@@ -0,0 +1,87 @@
+<!--Adopt-a-Dog checkout Page 4 -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-a-Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!--this is for the future style sheet-->
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+        <img src="adoptadog-logo.svg" width="500" alt="white dog silhoutte">
+        <h2>Seattle's Premier Dog Adoption Agency</h2>
+        <ing src="cart-icon.svg" width="200" alt="cart icon">
+          <a href="dog-checkout-page4.html">Checkout</a>
+      </header>
+      <nav>
+        <ul>
+          <li>Home</li>
+          <li>Dogs</li>
+          <li>Blog</li>
+        </ul>
+      </nav>
+         <form method="POST">
+           <label> Name:
+             <input type="text" name="applicantName">
+           </label>
+           <label> Address:
+             <input type="text" name="applicantAddress">
+           </label>
+           <label> City:
+             <input type="city" name="applicantCity">
+           </label>
+           <label> State:
+             <select name="state"
+               <option value="Washington">Washington</option>
+               <option value="Oregon">Oregon</option>
+               <option value="Idaho">Idaho</option>
+               <option value="California">California</option>
+               <option value="Nevada">Nevada</option>
+               <option value="BC">British Columbia</option>
+             </select>
+           </label>
+           <label> Zipcode:
+             <input type="zipcode" name="applicantZipcode">
+           </label>
+           <label> City:
+             <input type="city" name="applicantCity">
```

Comment:
with your name attributes stick to one method of variable naming you have applicantCity and firsttime. For names I like kebab casing where you split words up with a `-` so it would be `applicant-city` and `first-time`. But it's a preference, just as long as it's consistent. In the real world it's a little more important, just a heads up.